_includes/home/_content-menu.html

```diff
@@ -0,0 +1,15 @@
+<div class="container">
+    <div class="col-md">
```

Comment:
Where is `col-md` defined? That looks like a BS4 selector but this site uses BS3.